lib/iris/tests/unit/fileformats/grib/test_GribWrapper.py

```diff
@@ -118,7 +117,7 @@ def test_regular_sequential(self):
         for i, _ in enumerate(tell_tale):
             gw = GribWrapper(grib_message, grib_fh)
             self.assertTrue(is_lazy_data(gw._data))
-            proxy = gw._data.concrete
+            proxy = as_concrete_data(gw._data)
             self.assertIsInstance(proxy, GribDataProxy)
             self.assertEqual(proxy.shape, (10, 20))
```

Comment:
These tests are failing. It looks like `gw._data.concrete` should return the proxy rather than realising the data.
@corinnebosley is looking into this test failure.